run.cmd

```diff
@@ -37,6 +37,11 @@ call "%_VSCOMNTOOLS%\VsDevCmd.bat"
 :: misleading value (such as 'MCD' in HP PCs) may lead to build breakage (issue: #69).
 set Platform=
 
+:: Disable telemetry, first time experience, and global sdk look for the CLI
```

Comment:
It'd be nice to disable these even higher - so they don't affect `init-tools`.  Is that possible?  Or would it need to be duplicated in both places?

Reply:
oh, I see init-tools is called right after this.... never mind.